sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetSchemaSuite.scala

```diff
@@ -902,6 +1890,181 @@ class ParquetSchemaSuite extends ParquetSchemaTest {
     """.stripMargin,
     writeLegacyParquetFormat = true)
 
+  testParquetToCatalyst(
+    "SPARK-36935: test case insensitive when converting Parquet schema",
+    StructType(Seq(StructField("F1", ShortType))),
+    """message root {
+      |  optional int32 f1;
+      |}
+      |""".stripMargin,
```

Comment:
indentation?

Reply:
Eh I just followed the previous test case on this..